app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for human in people:
+        person = Person(human["name"], human["age"])
+        if "wife" in human and human["wife"] is not None:
+            for human1 in people:
+                if human["wife"] == human1["name"]:
+                    person_wife = Person(human1["name"], human1["age"])
+                    person_wife.husband = person
+                    person.wife = person_wife
```

Comment:
Redundant iterations

You can create people list using list comprehension and after that use `Person.people` dict to retrieve human by name